src/frontend/Sider/Behandling/Vilkårvurdering/EndreVilkår.tsx

```diff
@@ -218,7 +228,19 @@ export const EndreVilkår: FC<EndreVilkårProps> = (props) => {
     const EndrePerioder = (
         <HStack gap="4" align="start">
             <FeilmeldingMaksBredde $maxWidth={152}>
-                {skalBrukeMånedÅrVelger ? (
+                {erMidlertidigOvernatting ? (
+                    <DateInputMedLeservisning
+                        label={'Fra'}
+                        value={fom}
+                        onChange={(dato) => {
+                            settFom(dato);
+                            settFeilmeldinger((prevState) => ({ ...prevState, fom: undefined }));
+                        }}
+                        readOnly={!props.alleFelterKanRedigeres}
+                        size="small"
+                        feil={feilmeldinger.fom}
+                    />
+                ) : skalBrukeMånedÅrVelger ? (
```

Comment:
Tror ikke dette skalerer så veldig godt 😅 Allerede litt vondt med to nøstede ternary-operatorer